npm-packages.md

```diff
@@ -96,6 +96,42 @@ Use [Semantic Release](https://github.com/semantic-release/semantic-release) in
 
 If your release branch is protected (a good idea) use [guardian/actions-merge-release-changes-to-protected-branch](https://github.com/guardian/actions-merge-release-changes-to-protected-branch) to commit version bumps.
 
+##### **Parsing Commit Messages**
+
+Use tooling to aid crafting and verfying commits and/or PR titles to ensure that the new version determined by the [semantic-release/commit-analyser](https://github.com/semantic-release/commit-analyzer) plugin is correct with one of the following stategies:
```

Comment:
```suggestion
Use tooling to help write and verify commits/PR titles. This will ensure that the [semantic-release/commit-analyser](https://github.com/semantic-release/commit-analyzer) plugin can determine the correct new version using one of the following strategies:
```